ice40/devices/top-routing-virt/CMakeLists.txt

```diff
@@ -31,8 +31,9 @@ define_device_type(
   DEVICE_TYPE top-routing-virt
   ARCH ice40
   ARCH_XML arch.xml
-  SCRIPT_OUTPUT_NAME timing
+  SCRIPT_OUTPUT_NAME timing tiles
```

Comment:
Is "tiles" required here?  It seems like only "UPDATE_TILES" is needed?

Reply:
Yeah, leftover

Reply:
Done